sdcm/sct_config.py

```diff
@@ -265,6 +265,9 @@ class SCTConfiguration(dict):
         dict(name="use_mgmt", env="SCT_USE_MGMT", type=boolean,
              help="When define true, will install scylla management"),
 
+        dict(name="skip_downstream_jobs", type=boolean,
+             help="When define true, the job will not trigger the jobs in its downstream"),
```

Comment:
@bentsi Since this parameter is only relevant for the pipeline, and never in the code, do I even need to specify it in sct_config?

Reply:
I doubt

Reply:
I'll remove it, then.